### DISCUSSION TITLE
Lab-based alternatives to Real-world measurements: Clarify the role of measurement

### DISCUSSION THREAD

**Discussion by dschien** [Green-Software-Foundation/sci#195] [SCI feedback]
There are several references to "measurement" in this section. For the majority of software applications measurements "in the wild" are impossible as they are distributed over several servers, networks and user devices and no party can physically take measurements on all parts of this supply chain.

I suggest substituting this term for something that takes this complication into account.

**Comment by atg-abhishek** [comment]
Do you have any suggestions on what the other phrasing would be?

**Comment by dschien** [comment]
Regarding the meaning of measurement. [L.J. Ferris. A new definition of measurement. 2004](https://reader.elsevier.com/reader/sd/pii/S0263224104000168) provides this definition: "‘Measurement is an empirical process, using an instrument, effecting a rigorous and objective mapping of an observable into a category in a model of the observable that meaningfully distinguishes the manifestation from other possible and distinguishable manifestations’’."

Key here are empirical and use of instrument.

Similarly [Roman Z. Morawski in Technoscientific Research](https://www.degruyter.com/document/doi/10.1515/9783110584066-006/html)
"... measurement is a “process of experimentally obtaining one or more quantity values that can reasonably be attributed to a quantity”. Note the experimental. 

I think this use of the term also comes across in Asim's latest addition to the introduction in [PR](https://github.com/Green-Software-Foundation/software_carbon_intensity/blob/dae26d17ebed84ef291e19340d45aa9a4cb70cdb/Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md) 
"Suppose, there is a lack of access, capability, or rights to the necessary real-world data. In that case, the SCI allows for data generated through modeling using best estimates instead." - here, a distinction is drawn between "real-world data" and data generated through modelling. ie..measurements are being contrasted to modelling.

So, the question is, if we really want to refer to measurement throughout the document.
